lib/proto/service.proto

```diff
@@ -109,11 +110,12 @@ message Exclude {
 
 // The TreeState is derived from the Zcash z_gettreestate rpc.
 message TreeState {
-    string network = 1; // "main" or "test"
-    uint64 height = 2;
-    string hash = 3;    // block id
-    uint32 time = 4;    // Unix epoch time when the block was mined
-    string tree = 5;    // sapling commitment tree state
+    string network = 1;     // "main" or "test"
+    uint64 height = 2;      // block height
+    string hash = 3;        // block id
+    uint32 time = 4;        // Unix epoch time when the block was mined
+    string saplingTree = 5; // sapling commitment tree state
```

Comment:
Doesn't this have implications for the Service on the other side of the wire?

Reply:
The service on the other side would need to have the same proto file I think, yes (Although I don't think the names matter). This is the latest from `zcash/lightwalletd`, at the time it was committed.

Reply:
OK...   so this isn't something you generated fresh..   I am going to make this a submodule, so we can just depend directly on upstream.